fallout/benchmarks/benchmark.yaml

```diff
@@ -0,0 +1,81 @@
+ensemble:
+  server:
+    node.count: 3
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: nebula
+        cloud.instance.type: m3.large
```

Comment:
Confirm with Fallout team whether this is the appropriate size for servers in this case

Reply:
Current size is ms2.small based on recommendations from the Fallout team